openpgp/packet/symmetrically_encrypted.go

```diff
@@ -178,7 +178,7 @@ const mdcPacketTagByte = byte(0x80) | 0x40 | 19
 
 func (ser *seMDCReader) Close() error {
 	if ser.error {
-		return errors.SignatureError("error during reading")
+		return errors.ErrMDCMissing
```

Comment:
Not quite sure about these `ErrMDCMissing` before the MDC check, since returning a `StructuralError` for malformed data is probably more accurate. However, to avoid breaking compatibility, for now I've kept a `SignatureError`.